packages/nimble-components/src/banner/tests/banner.stories.ts

```diff
@@ -121,11 +128,13 @@ export const _banner: StoryObj<BannerArgs> = {
         dismissButtonLabel: {
             name: 'dismiss-button-label',
             description:
-                'Set to a localized label (e.g. `"Close"`) for the dismiss button. This provides an accessible name for assistive technologies.'
+                'Set to a localized label (e.g. `"Close"`) for the dismiss button. This provides an accessible name for assistive technologies.',
+            control: { type: 'none' }
```

Comment:
Note: These metadata/args changes just result in the 'Localizable labels' section being the last row in Storybook. With dismissButtonLabel/toggle being unspecified in the args object below, Localizable Labels appeared in the middle (before dismissButton/toggle). Once I gave them values in args, that resulted in a 'Set object' button appearing in their Storybook row without the control.type=none changes.